stubs/redis/redis/commands/graph/node.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+
+class Node:
+    id: Any
+    alias: Any
+    label: Any
+    labels: Any
+    properties: Any
+    def __init__(
+        self, node_id: Any | None = ..., alias: Any | None = ..., label: Any | None = ..., properties: Any | None = ...
```

Comment:
It has `isinstance(label, list)`, so we should probably error if someone passes some other iterable:
```suggestion
        self, node_id: Any | None = ..., alias: Any | None = ..., label: str | list[str] | None = ..., properties: Any | None = ...
```